### ISSUE TITLE
Explorer.EXE and fontview.exe crash@0.76

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rakibtg/gdipp#48]
```
crash

```

Original issue reported on code.google.com by `aTyr...@gmail.com` on 25 Apr 2010 at 5:39

Attachments:
- [crash_dump.rar](https://storage.googleapis.com/google-code-attachments/gdipp/issue-48/comment-0/crash_dump.rar)

**Comment by GoogleCodeExporter** [comment]
```
Thanks for reporting. I'll be working on it.
```

Original comment by `crendk...@gmail.com` on 25 Apr 2010 at 7:43
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Should have been fixed in 0.8.0.
```

Original comment by `crendk...@gmail.com` on 26 May 2010 at 5:20
- Changed state: **Fixed**
